crates/pop-cli/src/commands/build/parachain.rs

```diff
@@ -22,9 +22,6 @@ pub struct BuildParachainCommand {
 	/// Parachain ID to be used when generating the chain spec files.
 	#[arg(short, long)]
```

Comment:
Do we still need those here as it is not a command anymore. Also perhaps a name change of this type as it is not a command anymore.

Reply:
Cannot point the `Option` fields above but some don't have to be anymore.

Reply:
No need for macros anymore, cleaned in https://github.com/r0gue-io/pop-cli/pull/394/commits/b0b5946973c0f24ff07ef0d539ecd381b33f8fc4